streamx-common/src/test/scala/com/streamxhub/streamx/common/fs/LfsOperatorTest.scala

```diff
@@ -21,47 +21,37 @@
 package com.streamxhub.streamx.common.fs
 
 
-import com.streamxhub.streamx.common.fs.LfsOperatorTest.outputDir
+import com.streamxhub.streamx.common.fs.LfsOperatorTest.withTempDir
 import org.apache.commons.codec.digest.DigestUtils
 import org.apache.commons.io.{FileUtils, IOUtils}
 import org.junit.jupiter.api.Assertions._
-import org.junit.jupiter.api.{AfterAll, AfterEach, BeforeEach, Test}
+import org.junit.jupiter.api.Test
 
 import java.io.{File, FileInputStream}
+import java.nio.file.{Files, Paths}
 import scala.language.implicitConversions
 
 /**
  * @author Al-assad
  */
 object LfsOperatorTest {
 
-  val outputDir = "LfsOperatorTest-output/"
```

Comment:
all test cases share the same folder would cause issues when run ut concurrently